### PR TITLE
scylla_install_image:add ssm-agent installation for arm

### PR DESCRIPTION
**Pull request by yaronkaikov**
In https://github.com/scylladb/scylla-machine-image/pull/334/commits/434795fcd0522457d843dfb695f46493330486d0
snap installation was removed and we started to use ssm-agent deb package. Since we are building AMI based on ARM as well. we need to support both arches

### CODE REVIEW

**Comment by yaronkaikov** [comment]
Verified with: https://jenkins.scylladb.com/view/master/job/scylla-master/job/releng-testing/job/ami/1325/

File: dist/debian/build_deb.sh [hunk 1]
```diff
@@ -81,6 +81,7 @@ pkg_install python3
 pkg_install python3-devel
 pkg_install python3-pip
 
+git config --global --add safe.directory "$BUILDDIR"/scylla-machine-image
```
Comment (bentsi) [inline dist/debian/build_deb.sh:84]: why did you add this?

Comment (yaronkaikov) [reply dist/debian/build_deb.sh:84]: https://github.com/scylladb/scylla-machine-image/runs/6108039418?check_suite_focus=true 

because of that...
Added a Ref in commit description

File: packer/scylla_install_image [hunk 2]
```diff
@@ -138,7 +138,10 @@ if __name__ == '__main__':
             pyver = '3'
 
             # install .deb version of ssm-agent since we dropped snapd version
-            run('curl -L -o /tmp/amazon-ssm-agent.deb https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/debian_amd64/amazon-ssm-agent.deb')
+            if platform.machine() == 'x86_64':
```
Comment (bentsi) [inline packer/scylla_install_image:141]:
```suggestion
    arch = platform.machine()
    match arch:
        case "x86_64":
            run('curl -L -o /tmp/amazon-ssm-agent.deb https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/debian_amd64/amazon-ssm-agent.deb')
        case "aarch64":
            run('curl -L -o /tmp/amazon-ssm-agent.deb https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/debian_arm64/amazon-ssm-agent.deb'
        case _:
            raise Exception("Unknown arch")

```

**Review by bentsi — APPROVED** [review]

**Comment by yaronkaikov** [comment]
Dropped from next due some syntax error in code. it was merged while verification still was in progress
